ibridges/cli/data_operations.py

```diff
@@ -155,6 +161,7 @@ def run_shell(session, parser, args):
                 overwrite=args.overwrite,
                 resc_name=args.resource,
                 dry_run=args.dry_run,
+                on_err = args.on_error,
```

Comment:
Perhaps we should parse the argument first? Then we can give a nice error instead of a stack trace.

Reply:
I am not sure what you mean here?!